client/lib/abtest/active-tests.js

```diff
@@ -73,4 +73,13 @@ module.exports = {
 		defaultVariation: 'showMonthly',
 		allowExistingUsers: true
 	},
+
+	signupCopy: {
```

Comment:
I'd recommend keeping the test's name pretty unique - it should be easily searchable (so that based on the test's name you can search for it in P2 or in code). `singupCopy` might not be unique enough :/